scripts/cddl/generate.js

```diff
@@ -13,54 +13,55 @@ const specParsed = parse5.parse(specContent)
 /**
  * format the result and sort pre content to their corresponding classes
  */
-const cddlContent = getCDDLNodes([specParsed]).reduce((val, node) => {
-    if (!val[node.nodeClass]) {
-        val[node.nodeClass] = []
-    }
-    val[node.nodeClass].push(node.content)
-    return val
-}, {})
+const cddlNodes = getCDDLNodes([specParsed]).reduce((val, node) => {
+  if (!val[node.nodeClass]) {
```

Comment:
As a note, adding whitespace or other reformatting changes to this kind of PR is unnecessary and makes it harder to review.

Reply:
Sorry, you are right and I should have done that in a separate commit. Let me know if you want to see it reverted / splitted.